LICENSE

```diff
@@ -352,6 +352,14 @@ This project includes the software: underscore.js
   Used under the following license: The MIT License (http://opensource.org/licenses/MIT)
   Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2013)
 
+This project includes the software: underscore.js:1.7.0
+  Available at: http://underscorejs.org
+  Developed by: DocumentCloud Inc (http://www.documentcloud.org/)
+  Inclusive of: underscore*.{js,map}
+  Version used: 1.7.0
+  Used under the following license: The MIT License (http://opensource.org/licenses/MIT)
+  Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2014)
```

Comment:
(c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors

See [underscore-min.js](https://github.com/apache/incubator-brooklyn/blob/master/usage/cli/src/main/resources/brooklyn/item-lister/statics/style/js/underscore-min.js)